File: 251127/연속되는 수 2/continuous-number2.py
n = int(input())
arr = [int(input()) for _ in range(n)]

count = [0]
c = 1
i = 0
num = arr[i]
while i < n - 1:
    if num == arr[i + 1]:
        c += 1
        i += 1
    else:
        count.append(c)
        c = 1
        num = arr[i + 1]
        i += 1

count.append(c)
print(max(count))

File: 260319/연속되는 수 2/continuous-number2.js
const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split('\n');

const n = Number(input[0]);
const arr = input.slice(1, n + 1).map(Number);

let count = 0, answer = 0;
for (let i = 0; i < arr.length; i++) {
    if (i === 0) {
        count = 1;
    }

    if (i >= 1 && arr[i] != arr[i - 1]) {
        count += 1;
    }
    answer = Math.max(count, answer)
}

console.log(answer);